modules/swagger-annotations/src/main/java/io/swagger/oas/annotations/media/DiscriminatorMapping.java

```diff
@@ -16,13 +16,19 @@
 
 package io.swagger.oas.annotations.media;
 
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
 /**
  * 
  *
  * 
  **/
-
-
+@Target({  })
+@Retention(RetentionPolicy.RUNTIME)
+@Inherited
```

Comment:
it's ok, possibly removing `@Inherited` which doesn't apply to anything but classes